static/src/javascripts/bootstraps/enhanced/notifications.js

```diff
@@ -0,0 +1,180 @@
+define([
+    'bonzo',
+    'qwery',
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config',
+    'common/utils/storage',
+    'common/utils/ajax',
+    'common/utils/template',
+    'common/utils/mediator',
+    'common/utils/robust',
+    'common/views/svgs',
+    'common/modules/user-prefs',
+    'text!common/views/ui/notifications-subscribe-link.html',
+    'lodash/collections/some',
+    'lodash/arrays/uniq',
+    'lodash/arrays/without'
+], function (
+    bonzo,
+    qwery,
+    bean,
+    fastdom,
+    $,
+    config,
+    storage,
+    ajax,
+    template,
+    mediator,
+    robust,
+    svgs,
+    userPrefs,
+    subscribeTemplate,
+    some,
+    uniq,
+    without
+) {
+    var modules, reg, sub,
+        isSubscribed = false;
+
+    modules = {
```

Comment:
@sndrs Cos I'm a crap javascript coder and I just ripped off the liveblog bootstrap which works in the same way. :( Point taken about the svgs. Firefox - yes it does but that's more work so we've decided just to start with chrome. Hopefully ff will come later ..